src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -19,6 +20,16 @@ public class GeoMasterDataProvider : DiagnosticDataProvider, IDiagnosticDataProv
         private readonly IGeoMasterClient _geoMasterClient;
         private GeoMasterDataProviderConfiguration _configuration;
         private string _geoMasterHostName;
+        private Lazy<HttpClient> _scmClient = new Lazy<HttpClient>(() =>
+        {
+            var httpClient = new HttpClient();
+            httpClient.DefaultRequestHeaders.Accept.Clear();
+            httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue(HeaderConstants.OctetStreamContentType));
+            httpClient.DefaultRequestHeaders.Add(HeaderConstants.UserAgentHeaderName, "appservice-diagnostics");
```

Comment:
Do we have a constant for useragent that we use elsewhere (e.g.. Kusto data provider)? If not, may be we should create one.

Reply:
Yes, it's used at multiple places. I will create one and use it.

Reply:
Updated